imap_processing/ultra/tests/unit/data_path_fixtures.py

```diff
@@ -0,0 +1,67 @@
+"""Pytest plugin module for test data paths"""
+import sys
+from pathlib import Path
+
+import pytest
+
+
+@pytest.fixture()
+def ccsds_path():
+    """Returns the ccsds directory.
+    """
+    return Path(sys.modules[__name__.split(
+        '.')[0]].__file__).parent / 'ultra' / 'tests' / 'test_data' / 'l0' / \
+        'Ultra45_EM_SwRI_Cal_Run7_ThetaScan_20220530T225054.CCSDS'
```

Comment:
```suggestion
    filename = "Ultra45_EM_SwRI_Cal_Run7_ThetaScan_20220530T225054.CCSDS" #  to avoid line too long error
    return f"{imap_module_directory}/ultra/tests/test_data/l0/{filename}"
```